### ISSUE TITLE
ttab command opens new tab but no command prompt (Mac)

### ISSUE THREAD

**Issue by lc-eu2cloud** [mklement0/ttab#45]
Hi, 

I apologize if this has already been resolved. I can use ttab just fine. The only thing that bothers me is that I have to use Ctrl-C to return to the command prompt after I open tab in current window. Otherwise, it just hangs. 

Thanks for all the effort put into this. It's much appreciated :)

**Comment by mklement0** [comment]
Thank you for the nice feedback, @lc-eu2cloud.

Can you provide more details about your problem? What OS and specific version? Can you provide a sample command that exhibits the problem?

**Comment by lc-eu2cloud** [comment]
Sure thanks for the prompt reply!
Mac OS Mojave Version 10.14.6
I simply run "ttab" 
It opens a new tab in same window, but doesn't return to the command prompt so that I can enter commands. I hope that description helps

**Comment by mklement0** [comment]
Unfortunately, the oldest macOS version I have access to is 10.15 (Catalina),
and I don't see the problem there, neither in Terminal.app nor in iTerm2 3.4.16.

Which terminal application are you using and is there something unusual about your configuration?

**Comment by lc-eu2cloud** [comment]
I think I figured out the problem. I had uninstalled a version of node.js via homebrew, but didn’t remove it from my bash profile. Thus it got hung up on trying to read from the location where I installed node.js.

I will reinstall ttab tomorrow & see if it works. It’s also possible my macOS mojave version is not compatible with latest node.js & npm.

I appreciate you getting back to me & will keep you updated.
